docs/examples/Example_data_fusion_from_sensors.py

```diff
@@ -0,0 +1,463 @@
+
+# loop over the various timesteps
+for t in range(number_of_steps):
+    detections_1 = next(g1)
+    s1_detections.extend(detections_1[1])
+    detections_2 = next(g2)
+    s2_detections.extend(detections_2[1])
+    full_detections.extend(detections_1[1])
+    full_detections.extend(detections_2[1])
+
+    for detections in [detections_1, detections_2]:
+
+        # Run the Kalman tracker
+        KF_tracker.detector = DummyDetector(current=detections)
+        KF_tracker.__iter__()
+        _, tracks = next(KF_tracker)
+        kf_tracks.update(tracks)
+        del tracks
+
+        # Run the extended Kalman filter
+        EKF_tracker.detector = DummyDetector(current=detections)
+        EKF_tracker.__iter__()
+        _, tracks = next(EKF_tracker)
+        ekf_tracks.update(tracks)
+        del tracks
+
+        # Run the Particle Tracker
+        PF_tracker.detector = DummyDetector(current=detections)
+        PF_tracker.__iter__()
+        _, tracks = next(PF_tracker)
+        pf_tracks.update(tracks)
+
+metric_manager.add_data({'KF_tracks': kf_tracks}, overwrite=False)
+metric_manager.add_data({'PF_tracks': pf_tracks}, overwrite=False)
+metric_manager.add_data({'EKF_tracks': ekf_tracks}, overwrite=False)
+
+truths = set(ground_truth_simulator.groundtruth_paths)
+metric_manager.add_data({'truths': truths,
+                         'detections': full_detections}, overwrite=False)
+
+# %%
+# 4) Plot the tracks and the track performances
+# ---------------------------------------------
+# We have obtained the tracks and the ground truths
+# from the various trackers. It is time
+# to visualise the tracks and load the metric manager
+# to evaluate the performances.
+
+plotter = Plotterly()
+plotter.plot_measurements(s1_detections, [0,2], marker= dict(color='blue'),
+                         measurements_label='Radar 1 detections')
+plotter.plot_measurements(s2_detections, [0,2], marker= dict(color='cyan'),
+                         measurements_label='Radar 2 detections')
+plotter.plot_tracks(kf_tracks, [0,2], line= dict(color='grey'), track_label='KF tracks')
+plotter.plot_tracks(ekf_tracks, [0,2], line= dict(color='black'), track_label='EKF tracks')
+plotter.plot_tracks(pf_tracks, [0,2], particle=False, line= dict(color='red'), track_label='PF tracks')
+plotter.plot_ground_truths(truths, [0,2])
+plotter.plot_sensors(sensor1_platform, [0,1], marker=dict(color='black', symbol='129', size=15),
+                     sensor_label='Fixed Platform') # 131 is a diamond cross
+
+plotter.plot_sensors(sensor2_platform, [0, 1], marker=dict(color='darkslategray', symbol='cross',
+                                                           size=15),
+                     sensor_label='Moving Platform')
+plotter.fig
+
+
+# Loaded the plotter for the various metrics.
+from stonesoup.plotter import MetricPlotter
+
+metrics = metric_manager.generate_metrics()
+
+graph = MetricPlotter()
+
+graph.plot_metrics(metrics, generator_names=['OSPA_KF_truths',
```

Comment:
When I attempt to build the docs page, I receive the following error:

Traceback (most recent call last):
  File "C:\developer\winpython\Stone-Soup\docs\examples\Example_data_fusion_from_sensors.py", line 457, in <module>
    graph.plot_metrics(metrics, generator_names=['OSPA_KF_truths',
  File "C:\developer\winpython\Stone-Soup\stonesoup\plotter.py", line 726, in plot_metrics
    self.combine_plots(metrics_to_plot, metrics_kwargs)
  File "C:\developer\winpython\Stone-Soup\stonesoup\plotter.py", line 862, in combine_plots
    artists.extend(axis.plot([_.timestamp for _ in metric_values],
  File "C:\Program Files\WinPython\python-3.9.4.amd64\lib\site-packages\matplotlib\axes\_axes.py", line 1607, in plot
    self.add_line(line)
  File "C:\Program Files\WinPython\python-3.9.4.amd64\lib\site-packages\matplotlib\axes\_base.py", line 2105, in add_line
    self._update_line_limits(line)
  File "C:\Program Files\WinPython\python-3.9.4.amd64\lib\site-packages\matplotlib\axes\_base.py", line 2127, in _update_line_limits
    path = line.get_path()
  File "C:\Program Files\WinPython\python-3.9.4.amd64\lib\site-packages\matplotlib\lines.py", line 1022, in get_path
    self.recache()
  File "C:\Program Files\WinPython\python-3.9.4.amd64\lib\site-packages\matplotlib\lines.py", line 663, in recache
    x = _to_unmasked_float_array(xconv).ravel()
  File "C:\Program Files\WinPython\python-3.9.4.amd64\lib\site-packages\matplotlib\cbook\__init__.py", line 1333, in _to_unmasked_float_array
    return np.asarray(x, float)
  File "C:\Program Files\WinPython\python-3.9.4.amd64\lib\site-packages\numpy\core\_asarray.py", line 102, in asarray
    return array(a, dtype, copy=False, order=order)
TypeError: float() argument must be a string or a number, not 'datetime.datetime'